v20.2/known-limitations.md

```diff
@@ -8,78 +8,64 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
-  ?column?
-------------
-         1
-~~~
+### Spatial support limitations
 
-PostgreSQL:
+CockroachDB supports efficiently storing and querying [spatial data](spatial-data.html), with the following limitations:
 
-~~~
- x
----
- 1
-~~~
+- Not all [PostGIS spatial functions](https://postgis.net/docs/reference.html) are supported.
 
-[Tracking Github Issue](https://github.com/cockroachdb/cockroach/issues/46563)
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/49203)
 
-### Privileges required to access to certain virtual tables differs from PostgreSQL
+- The `AddGeometryColumn` [spatial function](functions-and-operators.html#spatial-functions) only allows constant arguments.
 
-Access to certain virtual tables in the `pg_catalog` and `information_schema` schemas require more privileges than in PostgreSQL. For example, in CockroachDB, access to `pg_catalog.pg_types` requires the `SELECT` privilege on the current database, whereas this privilege is not required in PostgreSQL.
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/49402)
 
-[Tracking Github Issue](https://github.com/cockroachdb/cockroach/issues/43177)
+- The `AddGeometryColumn` spatial function only allows the `true` value for its `use_typmod` parameter.
 
-### Concurrent SQL shells overwrite each other's history
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/49448)
 
-The [built-in SQL shell](cockroach-sql.html) stores its command history in a single file by default (`.cockroachsql_history`). When running multiple instances of the SQL shell on the same machine, therefore, each shell's command history can get overwritten in unexpected ways.
+- CockroachDB does not support the `@` operator for [spatial features](spatial-features.html). Instead of using `@` in spatial expressions, we recommend using the inverse, with `~`. For example, instead of `a @ b`, use `b ~ a`.
 
-As a workaround, set the `COCKROACH_SQL_CLI_HISTORY` environment variable to different values for the two different shells, for example:
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/56124)
 
-{% include copy-clipboard.html %}
-~~~ shell
-$ export COCKROACH_SQL_CLI_HISTORY=.cockroachsql_history_shell_1
-~~~
+- CockroachDB does not yet support [`INSERT`](insert.html)s into the [`spatial_ref_sys` table](spatial-glossary.html#spatial-system-tables). This limitation also blocks the [`ogr2ogr -f PostgreSQL` file conversion command](https://gdal.org/programs/ogr2ogr.html#cmdoption-ogr2ogr-f).
 
-{% include copy-clipboard.html %}
-~~~ shell
-$ export COCKROACH_SQL_CLI_HISTORY=.cockroachsql_history_shell_2
-~~~
+    [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/55903)
+
+- CockroachDB does not yet support `DECLARE CURSOR`, which prevents the `ogr2ogr` conversion tool from exporting from CockroachDB to certain formats. To work around this limitation, export data first to CSV or GeoJSON format.
```

Comment:
Can you add DECLARE CURSOR also prevents QGIS from working.

Can you also add 3D (https://github.com/cockroachdb/cockroach/issues/53091), M (https://github.com/cockroachdb/cockroach/issues/53091), TRIANGLE (https://github.com/cockroachdb/cockroach/issues/56196), TINs (https://github.com/cockroachdb/cockroach/issues/56196), k-nearest-neighbours (https://github.com/cockroachdb/cockroach/issues/55227) are currently not supported?

Reply:
Also curves / multicurves / circularstrings (https://github.com/cockroachdb/cockroach/issues/56199)